WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewModel.swift

```diff
@@ -5,8 +5,41 @@ import Yosemite
 
 import class AutomatticTracks.CrashLogging
 
+protocol ReviewsViewModelOutput {
+    var isEmpty: Bool { get }
 
-final class ReviewsViewModel {
+    var dataSource: UITableViewDataSource { get }
+
+    var delegate: ReviewsInteractionDelegate { get }
+
+    var hasUnreadNotifications: Bool { get }
+
+    var shouldPromptForAppReview: Bool { get }
+
+    var hasErrorLoadingData: Bool { get set }
+
+    func containsMorePages(_ highestVisibleReview: Int) -> Bool
+}
+
+protocol ReviewsViewModelActionsHandler {
```

Comment:
Same question as above - a comment describing this protocol would be nice.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5834/commits/9b691b402c2e1578f2a5fa3d074793b12c868a04